nvflare/lighter/impl/master_template.yml

```diff
@@ -819,3 +880,553 @@ helm_chart_values: |
   persist: /home/nvflare
 
 
+azure_start_svr_sh: |
+  #!/usr/bin/env bash
+  
+  DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+  function report_status() {
+    status="$1"
+    if [ "${status}" -ne 0 ]
+    then
+      echo "$2 failed"
+      exit "${status}"
+    fi
+  }
+
+  function check_binary() {
+    echo -n "Checking if $1 exists. => "
+    if ! command -v $1 &> /dev/null
+    then
+      echo "not found. $2"
+      exit 1
+    else
+      echo "found"
+    fi
+  }
+
+  function prompt() {
+    local __default="$1"
+    read -p "$2" ans
+    if [[ ! -z "$ans" ]]
+    then
+      eval $__default="'$ans'"
+    fi
+  }
+
+  # parse arguments
+  while [[ $# -gt 0 ]]
+  do
+    key="$1"
+    case $key in
+      --config)
+        config_file=$2
+        shift
+      ;;
+    esac
+    shift
+  done
+  
+  RESOURCE_GROUP=nvflare_rg
+  VM_NAME=nvflare_server
+  VM_IMAGE=Canonical:0001-com-ubuntu-server-focal:20_04-lts-gen2:latest
+  VM_SIZE=Standard_B2ms
+  NSG_NAME=nvflare_nsgs
+  ADMIN_USERNAME=nvflare
+  PASSWORD="NVFl@r3_P@88"$RANDOM"w0rd"
+  DEST_FOLDER=/var/tmp/cloud
+  NIC_NAME=${VM_NAME}VMNic
+  SERVER_NAME={~~server_name~~}
+  FL_PORT=8002
```

Comment:
This can't be changed in the script because all client and console need to know that.  If users change it, clients and consoles have no way to know that.